### ISSUE TITLE
How can we adapt blocks for a curved back?

### ISSUE THREAD

**Issue by joostdecock** [freesewing/freesewing#370]
Cindy writes:

> As I've aged I've developed rounded forward shoulders which are accompanied by a wider upper back and a narrow upper chest.

She was looking for a way to have Breanna with a wider back (higher across back value) without making the front wider too.

This could be summarized as a general purpose pattern hack for people with a rounded back.
We don't really do anything like that currently. In Brian, we don't measure back and front individually. In Breanna we do, but don't really use that to determine the (initial) width from armhole pitch point to armhole pitch point.

This seems like a worthwhile thing to try to add, although it mightn't be trivial to figure out an elegant way to do this without breaking a bunch of patterns.

**Comment by AlfaLyr** [comment]
I think the only way to take into account the width of the back at the armhole pitch point (which is more or less the wider point) is to measure it. I've seen that "across back" measurement in several places, it doesn't seem to be uncommon. With the shoulder to shoulder, armhole pitch to armhole pitch and underarm to underarm values (which we get from the high bust or the chest circumference) we can get a quite precise map of the curvature of the back.

**Comment by joostdecock** [comment]
We used to have an `acrossBack` measurement. But we got rid of it because people got it wrong more often than not: https://freesewing.org/blog/core-1-8-jaeger-across-back/#the-across-back-measurement-is-no-more

I am, in general, reluctant to solve fitting problems by throwing more measurements at the problem. My concern is that adding more measurements each time we come across a problem will set us on a course towards an intractable tangle of measurements. 

I've been thinking about this a lot recently, because stepping away from measurements almost feels like an existential crisis for FreeSewing because, you know, it's what we do.  

### The fitting problems we've solved
I think that there are essentially two things that determine great fit:

 - Things that are easy to measure (think neck circumference)
 - Things that are easy to see (think hunched back)

It feels to me that the kind of fitting problems we are dealing with almost exclusively deal with the latter group. I can't remember the last time somebody complained about the neck opening not fitting right or something. We've got to the point where we know how to get that right. It is not a problem today.

### The fitting problems we haven't solved
Still, fitting problems remain, and they tend to fall into one of two categories:

 - Bodies don't adhere to the assumptions made by the pattern designer
 - The pattern needs *a hack* for a speficic thing (like a sway-back adjustment)

The first one can be fixed by a combination of being careful to not to make assumptions. But I also think that ultimately, some body types need a design that is made for them, rather than trying to adapt a different design.

The latter is the group where I was thinking about a lot recently.

### Using more than measurements alone

@AlfaLyr mentioned at some point that for shoulder slope, we could present people with a bunch of pictures and say *or your shoulders more like this, or more like that*. I really liked that idea because people know their own bodies, and it's very intuitive to pick something that is more like you then the next thing.

So what if apart from measurements, we also store other things the influence fit in the model/person? Things about posture (hunched back, hollow lower back, and so on) or bellies, or the shape of bums, or I don't know what.

This way, we can draft a block based on measurements plus additional info about the model that is hard to capture in measurements, but has a known solution in pattern design.

### Why would this be better?

Traditional pattern design comes up with a shape that is know to work on the fit model. Then grades that up down, keeping the shape.

We prioritize respecting a user's measurements over the shape of the pattern piece. That is fine for measurements that are obvious and easy to take. But we're now trying to capture more subtle information into measurements, and doing what we do, this will take higher priority than thinking *does this even look like a T-shirt* because nobody is doing a visual check. It's all code.

Which is why, I am now leaning towards making blocks that:

 - Is drafted to measurements that we know to be reliable
 - Has options to tweak the fit based on common cases of how bodies differ from one person to the other
 - Have those 'options' stored along the measurements in the person/model


Long story short: I think that measurements alone, ain't enough.

**Comment by joostdecock** [comment]
v2.7 includes two  changes that can make this happen:

 - The fact that measurments can now be something else than mm (so we could have a % for curved back or something
 - The `raise` feature in core allowing developers to communicate back to the end user.

As such I'm closing this. The actual implementation in different patterns is outside the scope of this issue and would better be tracked in dedicated issues.
